openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -677,6 +715,8 @@
     "entity-fetch-error": "Error while fetching {{entity}}",
     "entity-updating-error": "Error while updating {{entity}}",
     "feed-post-error": "Error while posting the message!",
+    "fetch-entity-permissions-error": "Unable to get permission for asset.",
+    "fetch-feed-error": "Error while fetching feeds!",
```

Comment:
Let's not use specific message as it will make translation file lenghty
use something like:

Below key is already present, Please update it for all the fee related messages
```
"fetch-entity-error": "Error while fetching {{entity}}",
```

Reply:
sure, Thanks for catching it!